account_statement_import_file_reconciliation_widget/wizards/account_statement_import.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Licence AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+
+from odoo import models
+
+
+class AccountStatementImport(models.TransientModel):
+    _inherit = "account.statement.import"
+
+    def import_file_and_reconcile_button(self):
+        """Process the file chosen in the wizard, create bank statement(s)
+        and jump directly to the reconcilition widget"""
+        result = self._import_file()
+        statements = self.env["account.bank.statement"].browse(result["statement_ids"])
+        statements.button_post()
```

Comment:
button_post deleted in this commit https://github.com/OCA/OCB/commit/4fb0933986534e31337aa1dce3478a67deea72de#diff-96c5c4fc954f9deb57474d052d9c59b68cc41b610205d342949727ece08ac404L414 in v16

Reply:
Hello, I just the branch, please check again, thank you